code/modules/antagonists/malf_ai/malf_ai_modules.dm

```diff
@@ -156,6 +158,7 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module/malf))
 		Obtaining control of the weapon will be easier if Head of Staff office APCs are already under your control."
 	cost = 130
 	one_purchase = TRUE
+	minimum_apcs = 15 // So you cant speedrun delta
```

Comment:
Reminder that head of staff apcs were specifically made to reduce cost of delta for the explicit purpose of letting ais speedrun it, maybe have them count as multiple apcs?

Reply:
the minimum apc requirement is to make it impossible to sit silently for 50 minutes and delta while having no visible signs of preparing for that. Will see how it goes and maybe do that tweak later